neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
```

Comment:
> is the difference noticeable? (I didn't do a side-by-side comparison)
yup, was noticeable for me at least in some cases

> We already have Pillow as a dependency
interesting, I will try out the PIL im.save code then and see if setting quality actually works then